packages/backend/src/open_payments/payment/incoming/routes.test.ts

```diff
@@ -126,23 +121,6 @@ describe('Incoming Payment Routes', (): void => {
       list: (ctx) => incomingPaymentRoutes.list(ctx),
       urlPath: IncomingPayment.urlPath
     })
-
-    test('returns 500 for unexpected error', async (): Promise<void> => {
-      const incomingPaymentService = await deps.use('incomingPaymentService')
-      jest
-        .spyOn(incomingPaymentService, 'getWalletAddressPage')
-        .mockRejectedValueOnce(new Error('unexpected'))
-      const ctx = setup<ListContext>({
-        reqOpts: {
-          headers: { Accept: 'application/json' }
-        },
-        walletAddress
-      })
-      await expect(incomingPaymentRoutes.list(ctx)).rejects.toMatchObject({
-        status: 500,
-        message: `Error trying to list incoming payments`
-      })
-    })
```

Comment:
are we removing this because the case is already covered by the general error handling middleware? I realize the rejects assertions here are no longer correct but I'm just trying to verify the new behavior is tested.